h/h_api/bulk_api/model/data_body.py

```diff
@@ -0,0 +1,90 @@
+"""Models representing the data modifying payloads."""
+
+from h.h_api.enums import DataType
+from h.h_api.model.json_api import JSONAPIData
+from h.h_api.schema import Schema
+
+
+class UpsertUser(JSONAPIData):
+    """The data to upsert a user."""
+
+    validator = Schema.get_validator("bulk_api/command/upsert_user.json")
+
+    @classmethod
+    def create(cls, _id, attributes):
+        """
+        Create an upsert user body.
+
+        :param _id: User id
+        :param attributes: User attributes
+        :return:
+        """
+        return super().create(DataType.USER, _id=_id, attributes=attributes)
```

Comment:
So if I want to create an upsert user command (or just the data part of an upsert user, which will later be wrapped in a command wrapper) I instantiate an instance of this class, which is better than creating my own dict directly. Decouples things better

Reply:
This is the pattern in general. To create these objects in code we use `create()` to instantiate them from raw data we use `__init__()`. 

The idea is the code never has to know the exact guts of the object. It's a little untrue, because if you are setting the attributes you need to know what they are.

As you've no doubt seen by now, these all get bundled up later in the `CommandBuilder` which clicks together the right classes for you as well.